### PR TITLE
Update docs for 1.25

### PR DESCRIPTION
**Pull request by taneyland**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/reference/clusterspec/snow.md [hunk 1]
```diff
@@ -142,11 +142,11 @@ the control plane nodes for kube-apiserver loadbalancing.
 ### controlPlaneConfiguration.taints
 A list of taints to apply to the control plane nodes of the cluster.
 
-Replaces the default control plane taint, `node-role.kubernetes.io/master`. The default control plane components will tolerate the provided taints.
+Replaces the default control plane taint (For k8s versions prior to 1.24, `node-role.kubernetes.io/master`. For k8s versions 1.24+, `node-role.kubernetes.io/control-plane`).. The default control plane components will tolerate the provided taints.
```
Comment (chrisnegus) [inline docs/content/en/docs/reference/clusterspec/snow.md:145]:
```suggestion
Replaces the default control plane taint. For k8s versions prior to 1.24, it replaces `node-role.kubernetes.io/master`. For k8s versions 1.24+, it replaces `node-role.kubernetes.io/control-plane`. The default control plane components will tolerate the provided taints.
```

Comment (chrisnegus) [inline docs/content/en/docs/reference/clusterspec/snow.md:145]:
```suggestion
Replaces the default control plane taint. For k8s versions prior to 1.24, it replaces `node-role.kubernetes.io/master`. For k8s versions 1.24+, it replaces `node-role.kubernetes.io/control-plane`. The default control plane components will tolerate the provided taints.
```

File: docs/content/en/docs/reference/clusterspec/vsphere.md [hunk 2]
```diff
@@ -150,11 +150,11 @@ creation process are [here]({{< relref "../vsphere/vsphere-prereq/#prepare-a-vmw
 ### controlPlaneConfiguration.taints
 A list of taints to apply to the control plane nodes of the cluster.
 
-Replaces the default control plane taint, `node-role.kubernetes.io/master`. The default control plane components will tolerate the provided taints.
+Replaces the default control plane taint (For k8s versions prior to 1.24, `node-role.kubernetes.io/master`. For k8s versions 1.24+, `node-role.kubernetes.io/control-plane`). The default control plane components will tolerate the provided taints.
```
Comment (chrisnegus) [inline docs/content/en/docs/reference/clusterspec/vsphere.md:153]:
```suggestion
Replaces the default control plane taint. For k8s versions prior to 1.24, it replaces `node-role.kubernetes.io/master`. For k8s versions 1.24+, it replaces `node-role.kubernetes.io/control-plane`. The default control plane components will tolerate the provided taints.
```

**Review by chrisnegus — CHANGES_REQUESTED** [review]
Looks good. Just one suggestion (x2) about rewording the new taints description. If you don't like my wording, maybe try something else.

**Review by vivek-koppuru — COMMENTED** [review]
Can we replace the 1.23 references with 1.25 here on this page? https://anywhere.eks.amazonaws.com/docs/reference/vsphere/vsphere-ovas/

**Review by vivek-koppuru — APPROVED** [review]

**Pull request merged**